PlaceRouteHierFlow/placer/ILP_solver.cpp

```diff
@@ -1848,6 +1849,10 @@ bool ILP_solver::FrameSolveILPSymphony(const design& mydesign, const SeqPair& cu
         collb[ind + j] = -infty; colub[ind + j] = 0;
       }
     }
+    collb[N_var - 1] = -infty;
```

Comment:
Is it better to store the Area x and y variables index? If we add more vars and N_var is changed.

Reply:
Yes. That would be better. I’ll do it in the next PR.